src/Common/FileSystem.cpp

```diff
@@ -94,5 +94,20 @@ std::vector<std::byte> FileSystem::ReadAll(const fs::path& path)
 
 	return data;
 }
+std::vector<std::string> FileSystem::GetAllFilePaths(const std::string& path, const std::string& ext)
+{
+	std::vector<std::string> paths;
+
+	for (auto& entry : fs::recursive_directory_iterator(FindPath("Audio")))
```

Comment:
Can place all referenced to the "Audio" path with calls to
```
 static inline fs::path AudioPath() { return "Audio"; }
```

Reply:
Perhaps this function should return paths relative to Audio.

---

The `All` in the function name is confusing to me, it's only returning audio files.